client-src/elements/chromedash-toast.ts

```diff
@@ -1,27 +1,24 @@
 import {LitElement, css, html} from 'lit';
 import {SHARED_STYLES} from '../css/shared-css.js';
+import {property} from 'lit/decorators.js';
 
 const DEFAULT_DURATION = 7000;
 
 class ChromedashToast extends LitElement {
```

Comment:
You can also replace the `customElements.define` at the bottom of the file with
```suggestion
@customElement('chromedash-toast')
class ChromedashToast extends LitElement {
```